modules/jdk/25/configure.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
+
+# Update securerandom.source for quicker starts (must be done after removing jdk 8, or it will hit the wrong files)
+JAVA_SECURITY_FILE=/usr/lib/jvm/java/conf/security/java.security
+SECURERANDOM=securerandom.source
+if grep -q "^$SECURERANDOM=.*" $JAVA_SECURITY_FILE; then
+    sed -i "s|^$SECURERANDOM=.*|$SECURERANDOM=file:/dev/urandom|" $JAVA_SECURITY_FILE
+else
+    echo $SECURERANDOM=file:/dev/urandom >> $JAVA_SECURITY_FILE
+fi
```

Comment:
Unfortunately, this after-the-fact modification of `java.security` will break the JEP-493-style `jlink` from the runtime image. The same could be achieved with adding `-Djava.security.egd=file:/dev/urandom` to the java launch command which doesn't have that problem. Has this been considered as an alternative approach to this?

Reply:
Agreed.